README.md

```diff
@@ -14,9 +14,10 @@
 
 ## Introduction
 
-This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks. You can read a detailed tutorial and explanation of the Checkly Terraform provider here:
+This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks. You can read detailed tutorials and explanations of the Checkly Terraform provider here:
 
-* [Managing Checkly checks with Terraform](https://blog.checklyhq.com/managing-checkly-checks-with-terraform/)
+1. [Managing Checkly checks with Terraform](https://blog.checklyhq.com/managing-checkly-checks-with-terraform/)
+2. [Scaling Puppeteer and Playwright on Checkly with Terraform](https://blog.checklyhq.com/scaling-puppeteer-playwright-on-checkly-with-terraform/)
 
 ## Supported resources
```

Comment:
hey @kjda, is this section up-to-date? is something still missing for the resources which are not marked?

Reply:
Sorry for being unclear - in the above I am referring to the "Supported resources" section.